cli/pkg/kctrl/cmd/package/available/values_schema.go

```diff
@@ -15,10 +15,11 @@ import (
 // DataValueProperty holds the details of each property under Carvel package.spec.valuesSchema.openAPIv3.properties.
 // The example of the schema could be found at: https://carvel.dev/kapp-controller/docs/latest/packaging/#package-1
 // From above example, we would have the following:
-//   DataValueProperty.Key = "namespace"
-//   DataValueProperty.Type = "string"
-//   DataValueProperty.Description = "Namespace where fluent-bit will be installed."
-//   DataValueProperty.Default = "fluent-bit"
+//
```

Comment:
Looks like my formatter added some pretty spaces in comments, happy to remove these as well if y'all think it is not needed

Reply:
Removed!